### PR TITLE
[11.0][MIG] mgmtsystem audit

### PR DESCRIPTION
**Pull request by jalzaga**
Module mgmtsystem_audit Migration to version 11.0 #200

### CODE REVIEW

**Comment by jalzaga** [comment]
@max3903 Could you please let me know about the problems of this PR?

File: mgmtsystem_audit/models/mgmtsystem_audit.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        return self.write({'state': 'done',
+                           'closing_date': fields.Datetime.now()})
+
+    def get_action_url(self):
+        """
+        Return a short link to the audit form view
+        eg. http://localhost:8069/?db=prod#id=1&model=mgmtsystem.audit
+        """
+
+        base_url = self.env['ir.config_parameter'].get_param(
```
Comment (ivantodorovich) [inline mgmtsystem_audit/models/mgmtsystem_audit.py]: Code review, no testing. But according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

> Remove group_ids field in ir.config_parameter records, **and read/write them with sudo() instead** (https://github.com/odoo/odoo/commit/d0ca2d115e8b40308e1ff83bf9e7cc874762fbff).

File: mgmtsystem_audit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Management System - Audit",
+    "version": "11.0.1.0.0",
+    "author": "Savoir-faire Linux, Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
+    "license": "AGPL-3",
+    "category": "Management System",
+    "depends": ['mgmtsystem_nonconformity','base_automation'],
```
Comment (ivantodorovich) [inline mgmtsystem_audit/__manifest__.py]: LINT is complaining:
> mgmtsystem_audit/__manifest__.py:10:43: E231 missing whitespace after ','

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Minor coments, otherwise ok



--

There's a test that's failing on travis and runbot, but it's a test for the `mgmtsystem_nonconformity` module. It's failing because it tries to call `case_open` of `mgmtsystem.action`, and that method doesn't exists anymore. I'll fix it later in another PR.

```
ERROR
ERROR: test_state_transition (odoo.addons.mgmtsystem_nonconformity.tests.test_nonconformity.TestModelNonConformity)
` Close and reopen Nonconformity
Traceback (most recent call last):
`   File "/home/travis/build/OCA/management-system/mgmtsystem_nonconformity/tests/test_nonconformity.py", line 66, in test_state_transition
`     'mgmtsystem_nonconformity.stage_open')
`   File "/home/travis/OCB-11.0/odoo/fields.py", line 980, in __set__
`     record.write({self.name: write_value})
`   File "/home/travis/build/OCA/management-system/mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py", line 242, in write
`     action.case_open()
` AttributeError: 'mgmtsystem.action' object has no attribute 'case_open'
FAILED
Module mgmtsystem_nonconformity: 0 failures, 1 errors
At least one test failed when loading the modules.
Message not found: 'Modules loaded.'
----------
```

**Comment by jalzaga** [comment]
done @ivantodorovich

File: mgmtsystem_audit/data/audit_automated_actions.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<odoo>
```
Comment (ivantodorovich) [inline mgmtsystem_audit/data/audit_automated_actions.xml:1]: I've just noticed there are some warnings about this xml file. I'm guessing these actions are not working properly. Can you check this?
```
2018-10-29 21:30:15,941 165 INFO openerp_test odoo.modules.loading: loading mgmtsystem_audit/data/audit_automated_actions.xml
2018-10-29 21:30:15,948 165 WARNING openerp_test odoo.models: ir.actions.server.create() includes unknown fields: body_html, condition, email_to, subject
2018-10-29 21:30:15,971 165 WARNING openerp_test odoo.models: base.automation.create() includes unknown fields: server_action_ids
2018-10-29 21:30:16,003 165 WARNING openerp_test odoo.models: ir.actions.server.create() includes unknown fields: body_html, condition, email_to, subject
2018-10-29 21:30:16,029 165 WARNING openerp_test odoo.models: base.automation.create() includes unknown fields: server_action_ids
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Great!! Thanks!! Excellent work

**Review by max3903 — APPROVED** [review]
LGTM

**Comment by max3903** [comment]
Superseeded by #280

**Pull request closed without merging**